tests/attacks/conftest.py

```diff
@@ -65,9 +65,9 @@ def _tabular_classifier_list(attack, clipped=True):
             return None
 
         return [
-            potential_classier
-            for potential_classier in classifier_list
-            if attack.is_valid_classifier_type(potential_classier)
+            potential_classifier
+            for potential_classifier in classifier_list
+            if all(t in type(potential_classifier).__mro__ for t in attack.estimator_requirements)
```

Comment:
I find this line a bit hard to read. this is just a suggestion and I'm not sure maybe if it's because I'm not so used to the Python way of doing things but would it not be simpler in terms of readability to have this line as a function instead? as in `attack.passes_estimator_requirements(potential_classifier)`?

Reply:
I agree. This is the Black style formatting (https://black.readthedocs.io/en/stable/the_black_code_style.html) that we are applying since ART 1.2. Some formatting can take time to get used to, but it is really consistent across all files.